draft-ietf-rats-corim.md

```diff
@@ -1300,6 +1300,35 @@ they are not required to use the same internal data structures.
 For example, it is expected that the resources used during the initialisation
 phase can be amortised across multiple appraisals.
 
+## Verifier Abstraction
+
+This document assumes Verifier implementations may differ.
+To facilitate description of normative Verifier behavior,
+this document uses abstract representation of Verifier internals.
+
+* Claim: A structure that has a key-value pairing.
+* Environment Measurement Tuple (EMT): A structure containing a set of environment
+Claims that describe a Target Environment and a set of measurement Claims that
+describe attributes of the Target Environment.
+* possible state: Claims that describe various alternative forms of a Target Environment.
+Reference Values Claims typically describe various possible states due to versioning,
+manufactruing practices, or supplier configuration options.
+* actual state: Claims that describe a Target Environment instance,
+given some constraining context, such as a point in time or a network connection.
+Endorsed Values and Evidence typically are Claims about actual state.
+* Group: The scope in which Evidence, Reference Values, Endorsed Values and Appraisal Policy
+are applied.
+An Attester may be composed of multiple components, where each component may
+represent a scope of appraisal.
+* Authority: Provenance linking a Claim to the entity that asserted it.
```

Comment:
_(non-native speaker comment)_

I am not sure I understand "Provenance" in and by itself.  Maybe: "Provenance statement" or "Provenance assertion".